src/__mocks__/follow.js

```diff
@@ -0,0 +1,6 @@
+export default (Object.follow = {
+  createdAt: '2019-08-05T22:39:22.490Z',
```

Comment:
@sengayire, can you please use a dynamic date instead of a static date. You can for example use: `createdAt: new Date()`.